src/main/java/io/supertokens/inmemorydb/queries/GeneralQueries.java

```diff
@@ -63,6 +66,17 @@ private static boolean doesTableExists(Start start, String tableName) {
         }
     }
 
+    // TODO(mihaly): maybe we could be doing something smart here (checking schema info, pragma, etc)
+    private static boolean doesColumnExists(Start start, String tableName, String colName) {
+        try {
+            String QUERY = "SELECT " + colName + " FROM " + tableName + " LIMIT 1";
+            execute(start, QUERY, NO_OP_SETTER, result -> null);
+            return true;
+        } catch (SQLException | StorageQueryException e) {
+            return false;
+        }
+    }
```

Comment:
I don't think we need to be doing this. Altering tables is a risk thing to do automatically without a proper migration process (which we don't have yet). Instead, we would tell users to do this manually. It is risky cause:
- The user which the dev gives us may not have the permission to alter tables. In that case, what will happen?
- There could be multiple versions of the core running and if we alter the table this way on start, then other, older versions (which have not yet been updated) would start to fail.
- The user (for self hosted ones) may be reading the table directly, and in that case, returning an extra column might break their logic.

Reply:
Yeah, that's something I was thinking about, but asking for a manual migration sounded even worse.